pandas/_libs/tslibs/timezones.pyx

```diff
@@ -100,7 +100,7 @@ cpdef inline object maybe_get_tz(object tz):
             tz = _dateutil_tzlocal()
         elif tz.startswith('dateutil/'):
             zone = tz[9:]
-            tz = _dateutil_gettz(zone)
+            tz = dateutil_gettz(zone)
```

Comment:
so would like to rename these to be more consistent, maybe
``get_timezone_from_dateutil`` (different PR of course)